autoload/lsp/lsp.vim

```diff
@@ -406,6 +406,8 @@ export def AddFile(bnr: number): void
     endif
   augroup END
 
+  doautocmd <nomodeline> User LSPAttached
+
```

Comment:
Can you update the help file with a description of this autocmd event?

Reply:
Done, let me know if you have any feedback on the wording in the docs.